README.md

```diff
@@ -1315,8 +1318,76 @@ features: {
 
 Events published by the widget. Subscribe to these events using [on](#onevent-callback-context).
 
+### ready
+
+Triggered when the widget is ready to accept user input for the first time. Returns a `context` object containing the following properties:
+
+- **controller** - Current controller name
+
+```javascript
+signIn.on('ready', function (context) {
+  if (context.controller === 'primary-auth') {
+    // The primary authentication form is ready for user input
```

Comment:
This example could imply that `ready` is emitted whenever a new controller is ready. To avoid confusion with `afterRender`, I'd suggest removing the `context.controller` example here.